test/adapters/level_zero/level_zero_memcheck.match

```diff
@@ -0,0 +1,2 @@
+{{IGNORE}}
+{{.*}} ERROR SUMMARY: {{[03]}} errors from {{[03]}} contexts {{.*}}
```

Comment:
I was forced to allow 3 leaks. They originate in libigc, so there's not much we can do here. This should go away once we update all our runners to a newer driver version.